Sources_v3/Workers/ChannelMemberListUpdater.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+/// Makes a channel members query call to the backend and updates the local storage with the results.
+class ChannelMemberListUpdater<ExtraData: ExtraDataTypes>: Worker {
+    /// Makes a channel members query call to the backend and updates the local storage with the results.
+    /// - Parameters:
+    ///   - query: The query used in the request.
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    func load(_ query: ChannelMemberListQuery, completion: ((Error?) -> Void)? = nil) {
+        let channelEndpoint: Endpoint<ChannelPayload<ExtraData>> = .channel(query: .init(cid: query.cid))
+        apiClient.request(endpoint: channelEndpoint) { channelResult in
```

Comment:
This API call is needed only when the channel doesn't exist locally yet, which will be very rare.

Reply:
fixed in 24ae4cbb7982f78158fb534cc23319c533ddd44c